features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/components/LastUpdatedTime.jsx

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import React from 'react';
+import PropTypes from 'prop-types';
+import moment from 'moment';
+import { Typography, Tooltip } from '@material-ui/core';
+import { FormattedMessage } from 'react-intl';
+
+/**
+ * API/Product last updated time.
+ * @param {Parameter} props last updated time.
+ * @returns {Property} The sum of the two numbers.
+ */
+function LastUpdatedTime(props) {
```

Comment:
Did we remove the last updated time element from the previous place? otherwise, it will appear twice

Reply:
yes. @erangatl will remove it and fix styling issues in topdetail header.

Reply:
eranga's PR - https://github.com/wso2/carbon-apimgt/pull/7582/files